README.md

```diff
@@ -1,2 +1,184 @@
 # next-common-files
+
+[travis-url]:https://travis-ci.org/moxystudio/next-common-files
+[travis-image]:http://img.shields.io/travis/moxystudio/next-common-files/master.svg
+[codecov-url]:https://codecov.io/gh/moxystudio/next-common-files
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/next-common-files/master.svg
+
 Next.js plugins that insert loaders for common files.
+
+Projects developed with Next.js have to manually insert rules in the configuration file for handling file types that are potentially very common across multiple projects.
+
+These plugins quicken the initial setup of a project by removing that effort from the process, demanding less time from the developer.
+
+
+## Installation
+
+```sh
+$ npm install --save-dev @moxy/next-common-files
+```
+
+
+## Usage
+
+For single usage:
+
+```js
+// next.config.js
+const { withRasterImages } = require('@moxy/next-common-plugins');
+
+module.exports = withRasterImages({
+    /* options */
+});
+```
+
+For using multiple plugins, you can use [`next-compose-plugins`](https://github.com/cyrilwanner/next-compose-plugins). The examples in this README will follow the `next-compose-plugins` structure.
+
+```js
+// next.config.js
+const { withRasterImages, withPlayback, withSVG } = require('@moxy/next-common-plugins');
+const withPlugins = require('next-compose-plugins');
+
+module.exports = (phase, nextConfig) => {
+    withPlugins([
+        [withRasterImages()],
+        [withPlayback()],
+        [withSVG()],
+    ]),
+};
+```
```

Comment:
I think that in the above example it should be: 

```js
module.exports = withPlugins([
    [withRasterImages()],
    [withPlayback()],
    [withSVG()],
]);
```

Because currently it is exporting a function that returns `undefined`.

Reply:
Thanks, you're right. This also makes it more consistent with the next-compose-plugin documentation that is linked above this.